docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -90,25 +100,15 @@ spec:
       inTreeModuleToRemove: mod_b
 ```
 
-The ModuleLoader pod will first try to unload the in-tree `mod_b` before loading `mod_a` from the ModuleLoader image.  
-When the ModuleLoader pod is terminated and `mod_a` is unloaded, `mod_b` will not be loaded again.
-
-### Image pull policy
-
-Many examples in this documentation use the kernel version as image tag for the ModuleLoader image name.  
-If that image tag is being overridden on the registry, the Kubernetes
-[default image pull policy](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting) that KMM
-honors can lead to nodes not pulling the latest version of an image.  
-Use `.spec.moduleLoader.container.imagePullPolicy` to configure the right
-[image pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images) for your ModuleLoader
-DaemonSets.
+The worker Pod will first try to unload the in-tree `mod_b` before loading `mod_a` from the kmod image.  
+When the worker Pod is terminated and `mod_a` is unloaded, `mod_b` will not be loaded again.
```

Comment:
Should we describe the potential risks as a reason no to load `mod_b` after `mod_a` is being removed? I expect some questions around that.